tests/NamespaceUsagesTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types = 1);
+
+namespace Spaze\PHPStan\Rules\Disallowed;
+
+use PHPStan\File\FileHelper;
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+final class NamespaceUsagesTest extends RuleTestCase
+{
+
+	protected function getRule(): Rule
+	{
+		return new NamespaceUsages(
+			new DisallowedNamespaceHelper(new FileHelper(__DIR__)),
+			[
+				[
+					'namespace' => 'Inheritance\Sub',
+					'message' => 'no sub',
+					'allowIn' => [
+						'src/disallowed-allowed/*.php',
+						'src/*-allow/*.*',
+					],
+				],
+				[
+					'namespace' => 'Waldo\Quux\Blade',
+					'message' => 'no blade',
+					'allowIn' => [
+						'src/disallowed-allowed/*.php',
+						'src/*-allow/*.*',
+					],
+				],
+			]
+		);
+	}
+
+
+	public function testRule(): void
+	{
+		// Based on the configuration above, in this file:
```

Comment:
Add the other comment lines to better explain what's expected, see the other test files.